### DISCUSSION TITLE
Add ability to hide the `Source Simulation` hierarchy

### DISCUSSION THREAD

**Discussion by amhatre** [atoti/atoti#630] [General]
## Description

Add ability to hide the "Source Simulation" dimension, hierarchy and level.

## Context

Existing ActivePivot projects do not have the dimension/hierarchy "Epoch/Branch" in the UI when the `.withEpochDimension` api is not configured in their AP config. Therefore, existing users are not used to seeing this hierarchy in the UI. When migrating an AP project to Atoti+, it will be nice to be able to keep the available hierarchies as before.
Also for users who do not plan on using the simulations feature, they probably don't need this hierarchy.

## Other information (if relevant)

The hierarchies object retrieved from the cube, does not show "Source Simulation", therefore I could not hide it with something like -

```
h = risk_cube.hierarchies
h["Source Simulation"].visible = False
```

**Comment by bloussou** [comment]
Duplicate of https://github.com/atoti/atoti/issues/466

**Comment by tibdex** [comment]
This is actually not a duplicate of #466. The **Source Simulation** hierarchy is special and cannot be retrieved through the Python API with `h["Source Simulation"]` (nor with `h["Epoch"]`).
